server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1292,6 +1293,13 @@ boolean canSuspendTablets() {
     ThreadPools.watchCriticalScheduledTask(context.getScheduledExecutor()
         .scheduleWithFixedDelay(() -> ScanServerMetadataEntries.clean(context), 10, 10, MINUTES));
 
+    // TODO - create new threadpool?
```

Comment:
I assume that this cannot go into the TabletGroupWatcher because it needs to look at more than one Tablet? I think using the general scheduled executor threadpool should be fine. However, we may want to consider bumping the default value of `GENERAL_THREADPOOL_SIZE` from `1` to something larger. Probably good for a separate PR.

Reply:
Yes, TGW doesn't work in this case because because the task needs to look at multiple tablets together and not just one tablet at a time. Also, the task likely doesn't need to run very often to look for mergeable ranges where as TGW runs frequently

Reply:
I opened https://github.com/apache/accumulo/issues/5405 for this as a possible follow on for bumping the thread pool size